model/group_impl.go

```diff
@@ -33,6 +36,39 @@ func (repo *EntRepository) CreateGroup(ctx context.Context, name string, descrip
 	return ConvertEntGroupToModelGroup(created), nil
 }
 
+func (repo *EntRepository) GetMembers(ctx context.Context, groupID uuid.UUID) ([]*User, error) {
+	members, err := repo.client.Group.Query().Where(group.IDEQ(groupID)).QueryUser().All(ctx)
+	if err != nil {
+		return nil, err
+	}
+	modelmembers := []*User{}
+	for _, member := range members {
+		modelmembers = append(modelmembers, ConvertEntUserToModelUser(member))
+	}
+	return modelmembers, nil
+}
+
+func (repo *EntRepository) PostMember(ctx context.Context, groupID uuid.UUID, userID uuid.UUID) (*User, error) {
+	_, err := repo.client.Group.Update().Where(group.IDEQ(groupID)).AddUserIDs(userID).Save(ctx)
```

Comment:
書き方の統一のやつと、下でSELECT文を発行して再度取得してくるのは効率が悪くて、以下のようにすれば同時にupdateしたUserを取ってこれます。
```suggestion
	created, err := repo.client.Group.
	    Update().
	    Where(group.IDEQ(groupID)).
	    AddUserIDs(userID).
	    Save(ctx)
```